### PR TITLE
Fixed page links and card for upcoming event

### PR DESCRIPTION
**Pull request by kimberlycao**
Depending on whether we want to display available and upcoming events as cards, I need to generalize the card component to reuse it. I also wanted to add a "join event" button once the event is live.

### CODE REVIEW

**Review by KyrelJerome — CHANGES_REQUESTED** [review]
Looks good, just might want to fix the merge conflicts then its good to merge.

File: deerhunt/src/App.js [hunk 1]
```diff
@@ -41,12 +41,15 @@ class App extends React.Component {
         <Switch>
           <Route
             path="/"
-            component={<HomeLoggedIn email={this.state.email} isloggedin={true}/>}
+            component={
+              <HomeLoggedIn email={this.state.email} isLoggedIn={true} />
```
Comment (KyrelJerome) [inline deerhunt/src/App.js:45]: HomeLoggedIn doesnt need the prop anymore

File: deerhunt/src/components/Countdown.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { Text } from "@chakra-ui/react";
+
+class CountDownTimer extends Component {
```
Comment (KyrelJerome) [inline deerhunt/src/components/Countdown.js:4]: DAANG you put that work in lol

File: deerhunt/src/components/ComingSoonCard.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  Avatar,
+  useColorModeValue,
+  Image,
+} from "@chakra-ui/react";
+import BattleCodeLogo from "../images/ComingSoonLogo.png";
+import MCSSLogo from "../images/MCSSLogo.png";
+import UTMRoboticsLogo from "../images/UTMRoboticsLogo.png";
+import CountDownTimer from "./Countdown";
+
+export default function ComingSoonCard() {
```
Comment (KyrelJerome) [inline deerhunt/src/components/ComingSoonCard.js:17]: This is good! We'll adapt this later to use the api to get the information you have here.

File: deerhunt/src/components/Countdown.js (hunk 2 above)
Comment (kimberlycao) [reply deerhunt/src/components/Countdown.js:4]: I borrowed this code...no one is writing their own countdown component 😉

**Review by KyrelJerome — APPROVED** [review]

**Review by SpaSmALX — APPROVED** [review]

**Pull request merged**